nautobot/docs/administration/permissions.md

```diff
@@ -46,3 +46,41 @@ Site.objects.filter(
 ### Creating and Modifying Objects
 
 The same sort of logic is in play when a user attempts to create or modify an object in Nautobot, with a twist. Once validation has completed, Nautobot starts an atomic database transaction to facilitate the change, and the object is created or saved normally. Next, still within the transaction, Nautobot issues a second query to retrieve the newly created/updated object, filtering the restricted queryset with the object's primary key. If this query fails to return the object, Nautobot knows that the new revision does not match the constraints imposed by the permission. The transaction is then rolled back, leaving the database in its original state prior to the change, and the user is informed of the violation.
+
+## Assigning Permissions
+
+Permissions can be applied directly to users or to groups of users.
+
+### Permissions Assigned to Individual Users
+
+Permissions can be related directly to users from the Admin UI or the API:
+
+| -                                                           | Admin UI | API |
+| ----------------------------------------------------------- | -------- | --- |
+| Superusers                                                  | Yes      | Yes |
+| Users with permission to add or change `users | permission` | No       | Yes |
```

Comment:
Did we discuss whether this discrepancy between UI and API should be considered a bug?

Reply:
I'll open an issue so we can discuss

Reply:
Opened issue #1839